src/ray/gcs/store_client/in_memory_store_client.cc

```diff
@@ -74,13 +65,11 @@ Status InMemoryStoreClient::AsyncMultiGet(
   auto result = absl::flat_hash_map<std::string, std::string>();
   auto table = GetTable(table_name);
   if (table != nullptr) {
-    absl::ReaderMutexLock lock(&(table->mutex_));
     for (const auto &key : keys) {
-      auto it = table->records_.find(key);
-      if (it == table->records_.end()) {
-        continue;
+      auto valueOpt = table->Get(key);
```

Comment:
`value_opt`

Reply:
it's discouraged to use suffix for `optional` and `statusor`
see reference: https://abseil.io/tips/181 section "Avoiding the _or Suffix"